lib/components/src/blocks/Source.tsx

```diff
@@ -2,7 +2,7 @@ import React, { FunctionComponent } from 'react';
 import { styled, ThemeProvider, convert, themes } from '@storybook/theming';
 import { EmptyBlock } from './EmptyBlock';
 
-import { SyntaxHighlighter } from '../syntaxhighlighter/syntaxhighlighter';
+import { SyntaxHighlighter } from '../syntaxhighlighter/lazy-syntaxhighlighter';
```

Comment:
It seems that -right now- this does NOT cause an extra async loaded entrypoint yet.

Needs investigation why that is. I suspect there's another direct dependency on `react-syntaxt-highlighter` somewhere.